packages/trader/src/Modules/SmartChartBeta/Components/LaunchModal/__tests__/launch-modal-info.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import LaunchModalInfo from '../launch-modal-info';
+import { render, screen } from '@testing-library/react';
+
+jest.mock('Assets/SvgComponents/launch/ic-chart-launch.svg', () => jest.fn(() => <div id='light'>Light Image</div>));
+describe('Launch Modal Info', () => {
+    it('should show light icon when isDarkMode is false', () => {
+        render(<LaunchModalInfo isDarkMode={false} />);
+        expect(screen.getByText('Light Image')).toBeInTheDocument();
+    });
+});
```

Comment:
```
import * as React from 'react';
import LaunchModalInfo from '../launch-modal-info';
import { render, screen } from '@testing-library/react';

jest.mock('Assets/SvgComponents/launch/ic-chart-launch-dark.svg', () => jest.fn(() => <div>Dark Image</div>));
jest.mock('Assets/SvgComponents/launch/ic-chart-launch.svg', () => jest.fn(() => <div >Light Image</div>));

describe('Launch Modal Info', () => {
    it('should show dark icon in dark mode', () => {
        render(<LaunchModalInfo is_dark_mode/>);
        
        expect(screen.getByText('Dark Image')).toBeInTheDocument();
    });
    
    it('should show light icon when is_dark_mode is false', () => {
    render(<LaunchModalInfo is_dark_mode={false} />);
    
    expect(screen.getByText('Light Image')).toBeInTheDocument();
    });
});
```
Will it work? I noticed that you have 2 test files for 1 component

Reply:
https://github.com/jestjs/jest/issues/10458

There is one open issue in jest . The same thing is with this component . If I mock two images/svg in one test file.  One overwrites the second one. That's why two test files were made.

Reply:
Oh, sorry, I didn't know